Gruntfile.js

```diff
@@ -77,8 +94,8 @@ module.exports = (grunt) => {
       },
     },
     eslint: {
-      target: SOURCES.concat(['./tests/**/*.ts', './tests/**/*.js']),
-      options: { fix: true },
+      target: SOURCES.concat('./tests'),
+      options: { fix: false },
```

Comment:
Please leave this as `true`. We should use overrides for where we disagree.

Reply:
This will fix formatter.js, which does not have anything to do with this PR ;)